smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/i18n/ProviderLogging.java

```diff
@@ -147,4 +147,8 @@ public interface ProviderLogging extends BasicLogger {
     @LogMessage(level = Logger.Level.WARN)
     @Message(id = 243, value = "Processing method '%s' annotated with @Acknowledgement(POST_PROCESSING), but may not be compatible with post-processing acknowledgement management. You may experience duplicate (negative-)acknowledgement of messages.")
     void postProcessingNotFullySupported(String methodAsString);
+
+    @LogMessage(level = Logger.Level.INFO)
```

Comment:
Should be debug - not very informative for regular users

Reply:
Done